src/clients/SnapshotGraphql.ts

```diff
@@ -511,38 +380,8 @@ export class Snapshot extends API {
   async getPendingProposals(start: Date, end: Date, fields: (keyof SnapshotProposal)[], limit = 1000) {
     return await this.fetchProposals({ start, end, fields, scoresState: SnapshotScoresState.Pending }, 0, limit)
   }
-}
 
-export function filterDelegationTo(delegations: Delegation[], space: string): Delegation[] {
-  if (delegations.length > 1) {
-    return delegations.filter((delegation) => delegation.space === space)
-  }
-
-  return delegations
-}
-
-export function filterDelegationFrom(delegations: Delegation[], space: string): Delegation[] {
-  if (delegations.length === 0) {
-    return []
-  }
-
-  const unique_delegations = new Map<string, Delegation>()
-
-  for (const deleg of delegations) {
-    if (unique_delegations.has(deleg.delegator)) {
-      if (unique_delegations.get(deleg.delegator)?.space !== space) {
-        unique_delegations.set(deleg.delegator, deleg)
-      }
-    } else {
-      unique_delegations.set(deleg.delegator, deleg)
-    }
-  }
-
-  return Array.from(unique_delegations.values())
-}
-
-export async function fetchAndFilterDelegates(query: string, variables: any): Promise<DelegationResult> {
-  try {
+  async fetchDelegates(query: string, variables: any) {
```

Comment:
This one can be in another client so we can have a SnapshotSubgraph service. (We can do this later)